Makefile

```diff
@@ -16,23 +16,35 @@ install-rust-coverage:
 	cargo install rustfilt coverage-prepare
 	rustup component add llvm-tools-preview
 
+.PHONY: install-pgo
+	rustup component add llvm-tools-preview
+
 .PHONY: build-dev
 build-dev:
 	@rm -f python/pydantic_core/*.so
-	cargo build --features extension-module
-	@rm -f target/debug/lib_pydantic_core.d
-	@rm -f target/debug/lib_pydantic_core.rlib
-	@mv target/debug/lib_pydantic_core.* python/pydantic_core/_pydantic_core.so
+	pip install -v -e . --config-settings=build-args='--profile dev'
```

Comment:
For consistency I changed all of the `build-x` targets to use `pip install`, which is better than `cargo build` because it'll use `maturin` to build in exactly the same way as CI will. The speed difference should be minimal.

Reply:
(I like `maturin develop` personally but @samuelcolvin and @adriangb have been issues getting it to work, perhaps to debug another time.)